data/draft.js

```diff
@@ -1,28 +1,63 @@
-import {draft} from '../config/mongoCollections.js';
-import {data_validation, validateId} from './data_validation.js';
+import {draft, users} from '../config/mongoCollections.js';
+import {getUser} from "./users.js";
```

Comment:
`getUser` is not an actual function from `users.js`.
Use `import userFunctions from "./users.js";` with `userFunctions.getUserById` or `userFunctions.getUserByName` instead.